src/actions/actionTypes.js

```diff
@@ -30,3 +30,8 @@ export const POI_FORM_LINK_REMOVED = 'POI_FORM_LINK_REMOVED'
 export const POI_FORM_LINK_MODIFIED = 'POI_FORM_LINK_MODIFIED'
 export const POI_FORM_MEDIA_ADDED = 'POI_FORM_MEDIA_ADDED'
 export const POI_FORM_MEDIA_REMOVED = 'POI_FORM_MEDIA_REMOVED'
+
+// copy-paste
+export const POI_COPIED = 'POI_COPIED'
+export const POI_FORM_POI_PASTED = 'POI_PASTED'
```

Comment:
although it doesn't _really_ matter, it's generally good practice to name the action the same as its value